src/commands/pack/tarballs.ts

```diff
@@ -11,20 +11,16 @@ This can be used to create oclif CLIs that use the system node or that come prel
 
   static flags = {
     root: Flags.string({char: 'r', description: 'path to oclif CLI root', default: '.', required: true}),
-    targets: Flags.string({char: 't',
-      description: 'comma-separated targets to pack (e.g.: linux-arm,win32-x64)',
-      default: Tarballs.TARGETS.join(','),
-    }),
-    xz: Flags.boolean({description: 'also build xz', allowNo: true, default: true}),
-    tarball: Flags.string({ char: 't', description: 'optionally specify a path to a tarball already generated by NPM', required: false })
+    targets: Flags.string({char: 't', description: 'comma-separated targets to pack (e.g.: linux-arm,win32-x64)'}),
```

Comment:
I would prefer to not remove the `default` here since that would be a breaking change

It might be a better solution to change [this line](https://github.com/oclif/oclif/blob/main/src/tarballs/config.ts#L54) to prefer `updateConfig.node.targets` over `options.targets`. However, that's not a great solution either since user's would never be able to override what's the configured targets

@peternhale @RodEsp Do you have any opinions on this? It looks like the `default` for the `targets` flag was added soon after the `oclif-dev` commands were migrated into `oclif`: https://github.com/oclif/oclif/pull/458

Reply:
@mdonnalley Why is this a ״breaking change״?
Correct me if i'm wrong but it seems like if `flags?.targets?` is `undefined` the following line takes `TARGETS` array as a fallback
https://github.com/oclif/oclif/blob/71c3690ea20c333c5cacc2c248510263a80531dd/src/tarballs/config.ts#L70

Reply:
This appears to be the correct; command flag then config then default.